client/src/app/videos/+video-watch/video-watch.module.ts

```diff
@@ -12,14 +12,16 @@ import { NgbTooltipModule } from '@ng-bootstrap/ng-bootstrap'
 import { RecommendationsModule } from '@app/videos/recommendations/recommendations.module'
 import { VideoWatchPlaylistComponent } from '@app/videos/+video-watch/video-watch-playlist.component'
 import { QRCodeModule } from 'angularx-qrcode'
+import { InputSwitchModule } from 'primeng/inputswitch'
 
 @NgModule({
   imports: [
     VideoWatchRoutingModule,
     SharedModule,
     NgbTooltipModule,
     QRCodeModule,
-    RecommendationsModule
+    RecommendationsModule,
+    InputSwitchModule
```

Comment:
I think you'll have issues when you'll load your account page (after a watch page load), since we already import this module in the my account module.

Reply:
I didn't notice errors there.